### DISCUSSION TITLE
tailwind css is damn good

### DISCUSSION THREAD

**Discussion by ygndotgg** [tailwindlabs/tailwindcss#12360] [General]
I began using Tailwind CSS two months ago, and I'm really enthusiastic about learning more. I believe it's the best CSS framework currently available, and I'm eagerly anticipating the latest features in Tailwind CSS.

**Comment by RubenLien** [comment]
Transitioning from traditional CSS to Tailwind initially seemed like a daunting task, almost like scaling a steep mountain. However, after getting the hang of it, I must say the experience has been eye-opening. Everything feels more streamlined now. This journey has truly demonstrated that the initial effort was incredibly worthwhile!

**Comment by Octember** [comment]
Huge fan, where can I find open feature requests? Would like to contribute OSS
